web/admin/src/App.css

```diff
@@ -612,12 +656,11 @@ button {
 
 .multiline-editor__input {
   height: 5.8em;
-  width: 480px;
-  border: 1px solid #AEAEAE;
-  border-radius: 4px;
+  width: 100%;
   background-color: #FFFFFF;
   font-size: 1.25em;
-  padding: 10px 10px 10px 10px;
+  border: 0;
+  padding: 10px 0px 10px 0px;
```

Comment:
padding, margin, etc. have shortcuts when  left/right are the same and top/bottom are the same:

e.g. `padding: 10px 0;` is equivalent to `padding: 10px 0 10px 0;`